content/brazilian-portuguese/android-tictactoe/puzzle1.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Quebra-cabeça 1 - Execute o jogo e resolva-o"
+date: 2021-03-13T14:17:07.42-07:00
+draft: false
+weight: 2
+---
+
+## Execute o jogo
+Pressione o botão play (►) na parte superior do Android Studio para iniciar o jogo.
+<img src="../resources/_gen/images/run_app.png" height="30%" width="30%" title="Botão Executar aplicativo" alt="Botão Executar aplicativo"/>
+
+Você verá que o título na parte superior do aplicativo diz Toe-Tac-Tic (isso não parece certo!); vamos mudar para Tic-Tac-Toe.
+
+## Resolva o quebra-cabeça
+Clique duas vezes no arquivo `strings.xml` em app &rarr; res &rarr; valores:
+<img src="../resources/_gen/images/open_strings_file.gif" height="30%" width="30%" title="localização Strings.xml" alt="Mostra como navegar no toe strings.xml no Android Studio"/>
+
+Como você pode modificar este arquivo para dizer Tic-Tac-Toe?
```

Comment:
Awaiting instructions to translate or not "Tic-Tac-Toe".

Reply:
done